acceptance/allocator_terraform/load.tf

```diff
@@ -3,6 +3,11 @@
 variable "example_block_writer_instances" {
   default = 1
 }
+
+# Despite its name, example_block_writer actually has `photos` too. We're
+# sticking to this name for compatibility with Terrafarm. Since our load
```

Comment:
Interesting, didn't know terrafarm made assumptions on this - what could you point one out in the code?